onnx/defs/nn/defs.cc

```diff
@@ -2487,4 +2487,201 @@ ONNX_OPERATOR_SET_SCHEMA(
         )ONNX"
         ));
 
+static const char* LayerNormalization_ver16_doc = R"DOC(
+      This is layer normalization defined in ONNX as function.
+      The overall computation can be split into two stages.
+      The first stage is standardization, which makes the
+      normalized elements have zero mean and unit variances.
+      The computation required by standardization can be
+      described by the following equations.
+      ```
+      Mean = ReduceMean<axes=normalized_axes>(X)
+      D = Sub(X, Mean)
+      DD = Mul(Diff, Diff)
+      Var = ReduceMean<axes=axes>(DD)
+      VarEps = Add(Var, epsilon)
+      StdDev = Sqrt(VarEps)
+      InvStdDev = Reciprocal(StdDev)
+      Normalized = Mul(D, InvStdDev)
+      ```
+      where `normalized_axes` is `[axis, ..., rank of X - 1]`.
+      The variables `Var` and `StdDev` stand for variance and
+      standard deviation, respectively. The second output is
+      `Mean` and the last one is `InvStdDev`.
+      Depending on `stash_type` attribute, the actual computation may
+      happen in different floating-point precision.
```

Comment:
Regarding the use of "may": would it be possible to be more precise about what is guaranteed/required (in terms of precision) and what is allowed to be dependent on the implementation?

Reply:
Unless there is a good reason, it is better to be specific.

Reply:
Makes sense. I change it to `must`.